portal/README.md

```diff
@@ -22,6 +22,7 @@ This container listen to port 8000. For redirection configuration, please refer
 ## Setup environment variable
 
 We need to set up the environment variables:
+- `AUTHGEAR_APP_ID=accounts`
```

Comment:
It is better to just tell the reader to copy either `.env.example` or `.env.example.k8s`

Reply:
Sure, let me delete the whole list of environment variables in portal README.